Makefile

```diff
@@ -360,7 +360,8 @@ go-lint: check-git deps $(GOLANGCI_LINT) $(FAILLINT)
 	$(call require_clean_work_tree,'detected not clean work tree before running lint, previous job changed something?')
 	@echo ">> verifying modules being imported"
 	@# TODO(bwplotka): Add, Printf, DefaultRegisterer, NewGaugeFunc and MustRegister once exception are accepted. Add fmt.{Errorf}=github.com/pkg/errors.{Errorf} once https://github.com/fatih/faillint/issues/10 is addressed.
-	@$(FAILLINT) -paths "errors=github.com/pkg/errors,\
+	@$(FAILLINT) -paths "errors=github.com/thanos-io/thanos/pkg/errors,\
+github.com/pkg/errors=github.com/thanos-io/thanos/pkg/errors,\
```

Comment:
I think we should replace all the instances of /pkg/errors in a separate PR instead of unnecessarily bloating this one. 
```shell
💥➜ thanos git:(pkg/errors) ✗ make lint 2>&1 | wc -l
144
```
Your views @matej-g?